code/datums/station_traits/negative_traits.dm

```diff
@@ -765,4 +765,17 @@
 	report_message = "Due to a mishap at the Robust Softdrinks Megafactory, some drinks may contain traces of ethanol or psychoactive chemicals."
 	trait_to_give = STATION_TRAIT_SPIKED_DRINKS
 
+/// We're pulling a Jim Kramer with this one boys
+/datum/station_trait/gmm_spotlight
+	name = "GMM Economic Spotlight"
+	report_message = "This shift, the Galactic Mineral Market is doing a showcase on your crew's affulence! Every paycheck, the station newscasters will alert the crew who has the most credits."
+	trait_type = STATION_TRAIT_NEGATIVE
+	trait_to_give = STATION_TRAIT_ECONOMY_ALERTS
+	weight = 1
+	cost = STATION_TRAIT_COST_LOW
+	show_in_report = TRUE
+
+	threat_reduction = 5
+	dynamic_threat_id = "GMM Econ Spotlight"
```

Comment:
This has no good reason reduce threat.

Reply:
Fair, I'm not key'd in on current threat balance so I don't mind